x-pack/filebeat/module/cylance/protect/test/generated.log-expected.json

```diff
@@ -0,0 +1,3452 @@
+[
+    {
+        "@timestamp": "2016-01-29T08:09:59.000Z",
+        "event.category": "ZoneAdd",
```

Comment:
We probably don't want to copy the `event.category` or `event.type` values directly from the vendor-specific ones since they won't usually align with the ECS-allowed values. In the meantime WDYT about moving all of the vendor-specific categorization to `event.action`?

Reply:
+1 on not using vendor values in event.category.

Reply:
Done. I'm mapping all to `event.action` (single value)